.github/workflows/publish.yml

```
@@ -266,16 +266,32 @@ jobs:
           name: vector-${{ env.VECTOR_VERSION }}-arm-unknown-linux-musleabi
           path: target/artifacts/vector*
 
-  build-x86_64-apple-darwin-packages:
-    name: Build Vector for x86_64-apple-darwin (.tar.gz)
-    runs-on: macos-latest-large
+  build-apple-darwin-packages:
+    name: Build Vector for Apple Darwin (.tar.gz)
```

```suggestion
    name: Build Vector for Apple Darwin ${{ matrix.architecture }} (.tar.gz)
```

Looking at the custom build run made me realize it'd be nice to have the arch in the job name.